mobile/lib/modules/backup/providers/backup.provider.dart

```diff
@@ -314,20 +325,18 @@ class BackupNotifier extends StateNotifier<BackUpState> {
       );
       assetsFromExcludedAlbums.addAll(assets);
     }
-
     Set<AssetEntity> allUniqueAssets =
         assetsFromSelectedAlbums.difference(assetsFromExcludedAlbums);
     var allAssetsInDatabase = await _backupService.getDeviceBackupAsset();
 
-    if (allAssetsInDatabase == null) {
-      return;
-    }
-
-    // Find asset that were backup from selected albums
     Set<String> selectedAlbumsBackupAssets =
         Set.from(allUniqueAssets.map((e) => e.id));
     selectedAlbumsBackupAssets
-        .removeWhere((assetId) => !allAssetsInDatabase.contains(assetId));
+        .removeWhere((assetId) => !allAssetsInDatabase!.contains(assetId));
```

Comment:
Shouldn't the null check on line 337-339 happen before this? Otherwise the app will crash if `allAssetsInDatabase` is null?

Reply:
huh you know you have a good point, let me fix that

Reply:
@alextran1502 should be fixed now